clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -60,6 +60,8 @@
                                                                           ArrayOf.nullable(STRING),
                                                                           "An array of topics to fetch metadata for. If the topics array is null fetch metadata for all topics."));
 
+    public static final Schema METADATA_REQUEST_V2 = METADATA_REQUEST_V1;
```

Comment:
Please add a comment explaining that for the request v1 and v2 are the same, but the response for v2 is different.